vm/runtime.ts

```diff
@@ -1421,9 +1433,13 @@ module J2ME {
             r = fn.apply(this, arguments);
         }
         if (U) {
-          // TODO: Indicate unwinding on the method timeline.
-          if (methodType === MethodType.Native) {
-            ctx.leaveMethodTimeline(key, methodType);
+          if (methodInfo.isNative) {
+            // A fake frame that just returns is pushed so when the ctx resumes from the uwnind
+            // the frame will be popped triggering a leaveMethodTimeline.
+            var fauxFrame = Frame.create(null, []);
```

Comment:
*fauxFrame*, love the name. The comment is really helpful too.

Nit: uwnind -> unwind